src/infi/clickhouse_orm/fields.py

```diff
@@ -137,7 +137,7 @@ def to_python(self, value, timezone_in_use):
         raise ValueError('Invalid value for %s - %r' % (self.__class__.__name__, value))
 
     def to_db_string(self, value, quote=True):
-        return escape(timegm(value.utctimetuple()) or '0000000000', quote)
+        return escape(('0000000000' + str(timegm(value.utctimetuple())))[-10:], quote)
```

Comment:
Why not let python's string formatting do the zero padding? Something like '%010d' % timegm(...)